### DISCUSSION TITLE
[Icon Request] Add Ada and Odin icons

### DISCUSSION THREAD

**Discussion by JamesDSource** [ryanoasis/nerd-fonts#1024] [Ideas]
These are two relatively niche languages, but still have some industry use in embedded systems and game development respectively. Plus, I think they both look nice. 
![odin](https://user-images.githubusercontent.com/31216138/208268710-46d3aa0e-236d-41be-8261-4614adee5b8d.svg)
![ada](https://user-images.githubusercontent.com/31216138/208268746-210ea06a-9312-474b-acef-f84878b9a8fa.svg)

**Comment by chardoncs** [comment]
I was about to open a discussion about the Odin icon and saw this. Thank you, @JamesDSource !

**Comment by chardoncs** [comment]
I noticed that the Ada icon has already been added, but there is still no Odin icon.
